<launch>
  <arg name="config_file" default="$(find-pkg-share autoware_dummy_diag_publisher)/config/dummy_diag_publisher.param.yaml"/>
  <arg name="extra_config_file_sensor" default="$(find-pkg-share autoware_dummy_diag_publisher)/config/extra.param.yaml"/>
  <arg name="update_rate" default="10.0"/>

  <group>
    <node pkg="autoware_dummy_diag_publisher" exec="autoware_dummy_diag_publisher_node" name="dummy_diag_publisher" output="screen">
      <param from="$(var config_file)"/>
      <param from="$(var extra_config_file_sensor)"/>
      <param name="update_rate" value="$(var update_rate)"/>
    </node>
  </group>
</launch>
